extensions/amazon-aws/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Amazon AWS Changelog
 
+## [Step Functions command] - 2024-02-18
+
+- Added new command to list SQS queues ([#10801](https://github.com/raycast/extensions/pull/10801))
```

Comment:
```suggestion
- Added new command to find and open Step Functions ([#10801](https://github.com/raycast/extensions/pull/10801))
```

Reply:
oh my goodness, that was my bad copy&paste. Thanks, I fixed!